content/issues/2023-06-22-fedora-rawhide-testing-blocked-on-dnf5.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Fedora Rawhide Testing Blocked on Dnf5"
+date: "2023-06-22T09:30:00+02:00"
+affected:
+  - Testing Farm
+resolved: false
+section: issue
+severity: disrupted
+---
+
+dnf has been upgraded to dnf5 in Rawhide, resulting in various issues
+when running tests in Testing Farm.
+
+The Testing Farm team is working on fixing the issues
+and trying to resolve the situation ASAP.
+
+If you want to help, here are the issues in for DNF5 itself:
+
+- https://github.com/rpm-software-management/dnf5/issues/549
+- https://github.com/rpm-software-management/dnf5/issues/168
```

Comment:
```suggestion
The Testing Farm team together with dnf5 team is working on fixing the issues
and trying to resolve the situation ASAP.

Related DNF5 issues:

- https://github.com/rpm-software-management/dnf5/issues/549
- https://github.com/rpm-software-management/dnf5/issues/168
```

Reply:
I wanted to suggest people to contribute..;-)

Reply:
@lachmanfrantisek Could you also include the dnf5 team part from Laura's suggestion?

Reply:
Ah, never mind :slightly_smiling_face:

Reply:
I've included the first suggestion, but let the other as is if you don't mind.